util/file.py

```diff
@@ -473,3 +494,33 @@ def line_count(infname):
 def touch(fname, times=None):
     with open(fname, 'a'):
         os.utime(fname, times)
+
+def touch_empty(fname):
+    '''Make each file in fnames a zero-length file with the current timestamp.'''
+    with open(fname, 'w'):
+        pass
+
+def dump_file( fname, value ):
+    """store string in file"""
+    with open( fname, 'w' )  as out: 
+        out.write( str( value ) )
+
+def slurp_file( fname ):
+    """Read entire file into one string."""
+    with open_or_gzopen( fname ) as f:
+        return f.read()
+
+def tabjoin( *args, **kwargs ):
+    """Join args by tab"""
+    sep = kwargs.get( 'sep', '\t' )
+    return sep.join( map( str, args ) )
+
+def tabwriten( f, *args, **kwargs ):
+    """Write a tab-separated line to a file"""
+    f.write( tabjoin( *args, **kwargs ) )
+
+def tabwrite( f, *args, **kwargs ):
```

Comment:
at one point in history I think I had a nice tab file writer in here that I later removed as a part of purging unused code... that said, I feel like the normal way to do it is so terse in python anyway that it was less useful than the file reading functions were. Just a random comment, no request for changes here.